libs/providers/flagd/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@openfeature/flagd-provider",
+  "version": "0.0.1",
+  "type": "commonjs",
+  "scripts": {
+    "publish-if-not-exists": "cp $NPM_CONFIG_USERCONFIG .npmrc && if [[ $(npm show $npm_package_name@$npm_package_version version) = $(npm run current-version -s) ]]; then echo 'already published, skipping'; else npm publish --access public; fi",
+    "current-version": "echo $npm_package_version"
+  },
+  "dependencies": {
+    "@grpc/grpc-js": "^1.6.7",
```

Comment:
These dependencies must be added to the root package.json

Reply:
added

Reply:
You should also remove them from this `package.json`